client/src/app/site/pages/meetings/pages/motions/modules/categories/view-models/view-motion-category.ts

```diff
@@ -69,21 +74,6 @@ export class ViewMotionCategory extends BaseViewModel<MotionCategory> /* impleme
     public override getDetailStateUrl(): string {
         return `/${this.getActiveMeetingId()}/motions/categories/${this.sequential_number}`;
     }
-
-    /**
-     * @returns an array with all parents. The ordering is the direct parent
-     * is in front of the array and the "highest" parent the last entry. Returns
-     * an empty array if the category does not have any parents.
-     */
-    public collectParents(): ViewMotionCategory[] {
-        if (this.parent) {
-            const parents = this.parent.collectParents();
-            parents.unshift(this.parent);
-            return parents;
-        } else {
-            return [];
-        }
-    }
```

Comment:
Removed this since it is functionally equivalent to `allParents`, as far as i can see.